skbio/diversity/_fast_base_cy.pyx

```diff
@@ -0,0 +1,215 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+cimport numpy as np
+cimport cython
```

Comment:
You know ... maybe that doesn't [matter](http://docs.cython.org/0.14/src/tutorial/numpy.html)